apps/communications/contacts/test/unit/views/multiple_select_test.js

```diff
@@ -0,0 +1,95 @@
+/* global Contacts, MockContactsList, MockLazyLoader, MockMultipleSelectHTML */
+'use strict';
+
+requireApp('communications/contacts/js/views/multiple_select.js');
+requireApp('communications/contacts/test/unit/mock_contacts_list.js');
+require('/shared/test/unit/mocks/mock_lazy_loader.js');
+requireApp('communications/contacts/test/unit/mock_multiple_select.js.html');
+
+window.LazyLoader = window.LazyLoader || {};
+window.contacts = window.contacts || {};
+window.contacts.Matcher = window.contacts.Matcher || {
+  match: () => {}
+};
+window.navigator.mozContact = window.navigator.mozContact || {
+  save: () => {}
+};
+window.utils = window.utils || {};
+window.utils.status = window.utils.status || {
+  show: () => {}
+};
+
+suite('MultipleSelect view', function() {
+  var realLazyLoader,
+      realDocumentBody;
+
+  suiteSetup(function() {
+    realLazyLoader = window.LazyLoader;
+    window.LazyLoader = MockLazyLoader;
+    realDocumentBody = document.body.innerHTML;
+  });
+
+  suiteTeardown(function() {
+    window.LazyLoader = realLazyLoader;
+    document.body.innerHTML = realDocumentBody;
+  });
+
+  setup(function() {
+    document.body.innerHTML = MockMultipleSelectHTML;
+    Contacts.MultipleSelect.init();
+  });
+
+  test('Adding contacts to the view', function() {
+    Contacts.MultipleSelect.renderContacts(MockContactsList(), {});
+    assert.equal(document.querySelectorAll('li').length, 3);
+  });
+
+  test('Loading contacts from cursor', function() {
+    var howManyContacts = 4,
+        alreadyLoaded = 0,
+        onsuccessCallback;
+    Contacts.MultipleSelect.renderContacts([], {
+      set onsuccess(callback) {
+        onsuccessCallback = callback;
+        alreadyLoaded++;
+        onsuccessCallback({target: {result: {}}});
+      },
+      continue: () => {
+        if (alreadyLoaded++ < howManyContacts) {
+          onsuccessCallback({target: {result: {}}});
+        }
+      }
+    });
+
+    assert.equal(document.querySelectorAll('li').length, howManyContacts);
+  });
+
+  test('The view shows the filename', function() {
+    const FILENAME = 'FileName.vcf';
+    Contacts.MultipleSelect.renderContacts([], {}, FILENAME);
+    var title = document.getElementById('multiple-select-view-title');
+    assert.equal(title.textContent, FILENAME);
+  });
+
+
+  test('Importing all contacts', function() {
```

Comment:
I think this test is not necessary. Testing the importation should be done in an integration test